pkgs/tools/misc/digitemp/default.nix

```diff
@@ -0,0 +1,53 @@
+{ fetchFromGitHub, gnumake, lib, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "digitemp";
+  version = "3.7.2";
+
+  src = fetchFromGitHub {
+    owner = "bcl";
+    repo = "digitemp";
+    rev = "v${version}";
+    sha256 = "19zka5fcdxhhginaspak76l984iqq9v2j6qrwvi5mvca7bcj8f72";
+  };
+
+  nativeBuildInputs = [ gnumake ];
+
+  buildPhase = ''
+    runHook preBuild
+    make clean
+    make -j$NIX_BUILD_CORES LOCK=no ds9097 ds9097u
+    runHook postBuild
+  '';
+
+  installPhase = ''
```

Comment:
Use `-m555` for the executables.

Please copy in README/FAQ/etc into $out/share/doc/${pname} to provide some documentation.

You can also use `installShellFiles` to give you `installManPage` to copy in the provided man page.